src/kademlia/dht_tracker.cpp

```diff
@@ -155,9 +163,10 @@ namespace libtorrent { namespace dht
 		m_refresh_timer.expires_from_now(seconds(5), ec);
 		m_refresh_timer.async_wait(std::bind(&dht_tracker::refresh_timeout, self(), _1));
 
-		m_dht.bootstrap(m_state.nodes, f);
+		// bootstrap with mix of IP protocols via want/nodes/nodes6
+		m_dht.bootstrap(concat(m_state.nodes, m_state.nodes6), f);
```

Comment:
does the order matter? If not, you could save a bit by capturing the concatenated vector and pass it in by const reference to both calls. i.e.

```auto nodes = m_state.nodes;
nodes.insert(nodes.end(), m_state.nodes6.begin(), m_state.node6.end());
m_dht.bootstrap(nodes);
...
```